packages/lesswrong/components/common/Logo.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { registerComponent, getSetting } from 'meteor/vulcan:core';
+import { withStyles, withTheme } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  root: {
+    height: 48
```

Comment:
Can JP ask some dumb questions? What are the units for height? What does root mean in this context?

Reply:
Material UI's default unit is pixels, so `Number`s are cast to `Npx`

Root is just the classes I'm applying to the root component. As far as I can tell, it's an arbitrary name, but following a convention from other components